Modules/Utils/Sources/PocketCastsUtils/Sorting/PodcastSorter.swift

```diff
@@ -33,6 +37,25 @@ public enum PodcastSorter {
     public static func dateAddedSort(date1: Date, date2: Date) -> Bool {
         return date1.compare(date2) == .orderedAscending
     }
+
+    /**
+     Converts Chinese characters to their Pinyin equivalent
+     - Returns Pinyin string
+     */
+    private static func convertToPinyinIfNeeded(_ string: String) -> String {
```

Comment:
You can move this method to a `String` extension. This way we avoid adding an additional static method and it will be available in other places.

Reply:
Thank you for the insight! Converted to String extension.